Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Update the base image in Makefile when updating golang version. This has to
 # be pre-pulled in order to work on GCB.
-FROM golang:1.16.4 as build
+FROM golang:1.17 as build
```

Comment:
I think we should target minor version "1.17.0 like we did for "1.16.4".
Reason is that if there is a bug in 1.17 we know which MS versions are affected and can release new MS version with bumped golang version. Without this its not clear that we bumped patch version (for example switched to 1.17.1) as there not be any traces in git.